app/controllers/sessions_controller.rb

```diff
@@ -22,14 +22,14 @@ def verify_user
     if @user && @user.authenticate(params[:session][:password])
       login_successful
     else
-      flash[:failure] = "That login was unsuccessful"
+      flash[:failure] = "That login was unsuccessful."
```

Comment:
I'm not a huge fan of the language that came with this project for this flash message.

I'd prefer "Login unsuccessful." Not an essential change though.